src/item_factory.cpp

```diff
@@ -825,6 +825,19 @@ void Item_factory::finalize_post( itype &obj )
             }
         }
     }
+
+    // temperature_effects consistency
+    if( obj.temperature_effects_data ) {
+        for( const temperature_effect &eff : obj.temperature_effects_data->effects ) {
+            if( eff.temperature_above.has_value() == eff.temperature_below.has_value() ) {
+                debugmsg( "Exactly one of temperature_above and temperature_above must be set in %s temperature_effects",
+                          obj.id.str() );
+            }
+            if( !eff.transform_into.is_valid() ) {
+                debugmsg( "%s is not a valid item id (in %s temperature_effects)", obj.id.str() );
```

Comment:
If I'm not mistaken, the first `%s` should point to `eff.transform_into` string?

Reply:
Yes, thank you!
Fixed, and tested. Works now!